include/TradingEngine/order_status.h

```diff
@@ -0,0 +1,22 @@
+/*
+Contains declaration for OrderStatus enum and Order status map. 
+OrderStatus enum is a property of the Order class and Order
+Status map is used for string representation of Order info.
+*/
+
+#ifndef TRADING_ENGINE_INCLUDE_ORDER_STATUS_H_
+#define TRADING_ENGINE_INCLUDE_ORDER_STATUS_H_
+
+
+#include <unordered_map>
+#include <string>
+
+enum class OrderStatus {
+	NEW_ORDER,
+	PARTIALLY_FILLED,
```

Comment:
What does filled mean here? you mean matched or fulfilled?

Reply:
changed to FULFILLED